src/metrics/misc.ts

```diff
@@ -174,16 +164,60 @@ export class MetadataMetric implements Metric {
     const blackPlayer = metadata?.find((data) => data.startsWith('[Black'));
     const whiteUsername = whitePlayer?.split('"')[1];
     const blackUsername = blackPlayer?.split('"')[1];
-    
+
     if (whiteUsername) {
-      this.playerGameStats[whiteUsername] = (this.playerGameStats[whiteUsername] || 0) + 1;
+      this.playerGameStats[whiteUsername] =
+        (this.playerGameStats[whiteUsername] || 0) + 1;
     }
     if (blackUsername) {
-      this.playerGameStats[blackUsername] = (this.playerGameStats[blackUsername] || 0) + 1;
+      this.playerGameStats[blackUsername] =
+        (this.playerGameStats[blackUsername] || 0) + 1;
+    }
+
+    if (metadata[4] === '[Result "1-0"]') {
+      this.whiteWins++;
+    } else if (metadata[4] === '[Result "0-1"]') {
+      this.blackWins++;
+    } else if (metadata[4] === '[Result "1/2-1/2"]') {
+      this.ties++;
+    } else {
+      throw new Error('Invalid result');
     }
```

Comment:
tested this code, apparently PGN result is not always at index 4. going to change this in latest PR